packages/hash/external-services/docker-compose.yml

```diff
@@ -75,13 +75,39 @@ services:
     environment:
       HASH_GRAPH_PG_MIGRATION_URL: "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${HASH_GRAPH_PG_DEV_DATABASE}"
 
+  telemetry-collector:
```

Comment:
we can add this to the `hash_graph` docker compose too if we want it in isolation. I found it would be most useful for the external services one since errors would be more friendly to find for other folks